src/Datadog.Trace/Span.cs

```diff
@@ -200,6 +200,23 @@ public Span SetTag(string key, string value)
                         Log.Warning("Value {0} has incorrect format for tag {1}", value, Trace.Tags.Analytics);
                     }
 
+                    break;
+                case Trace.Tags.Measured:
+                    if (value?.ToBoolean() == true || string.IsNullOrEmpty(value))
```

Comment:
I think we should remove the `string.IsNullOrEmpty(value)` condition, so we only accept values can we can convert into `true` or `false`. The spec says:
>If `setTag("_dd.measured")` is called without a value, then the default value of `1` is used.

I don't think this applies to .NET since we don't have an overload for `setTag()` with no value. If a user writes the following lines, the intent is probably to _remove_ the tag, not to set it:
```csharp
// set the tag
span.SetTag(Trace.Tags.Measured, "1");

// remove the tag?
span.SetTag(Trace.Tags.Measured, null);
```

This is also consistent with every other tag, where setting `null` actually removes the tag. What do you think?

Reply:
Makes sense, thanks for the clarification! Addressed in commit **606b428**, please let me know if you have further notes.